support-frontend/assets/pages/supporter-plus-landing/components/threeTierCard.tsx

```diff
@@ -87,33 +85,81 @@ const checkmarkList = css`
 	}
 `;
 
+const benefitsPrefixCss = css`
+	${textSans.small()};
+	color: ${palette.neutral[7]};
+	text-align: left;
+	strong {
+		font-weight: bold;
+	}
+`;
+
+const benefitsPrefixPlus = css`
+	${textSans.small()};
+	color: ${palette.neutral[7]};
+	display: flex;
+	align-items: center;
+	margin: ${space[3]}px 0;
+	:before {
+		content: '';
+		height: 1px;
+		background-color: ${palette.neutral[86]};
+		flex-grow: 2;
+		margin-right: ${space[2]}px;
+	}
+	:after {
+		content: '';
+		height: 1px;
+		background-color: ${palette.neutral[86]};
+		flex-grow: 2;
+		margin-left: ${space[2]}px;
+	}
+`;
+
+const frequencyCopyMap = {
+	MONTHLY: 'month',
+	ANNUAL: 'year',
+};
+
+const priceSuffixCopy = (currency: string, planCost: PlanCosts) => {
```

Comment:
const priceSuffixCopy = (currency: string, planCost: PlanCosts) => {
	if (planCost.discount) {
		const duration = planCost.discount.duration.value;
		return `${currency}${planCost.discount.price} for the first ${
			duration > 1 ? duration : ''
		} ${frequencyCopyMap[planCost.discount.duration.period]}${
			duration > 1 ? 's' : ''
		}, then ${currency}${planCost.price}`;
	}
};

Reply:
maybe just little shorter?